### DISCUSSION TITLE
Cross-compilation - target: win, host: linux

### DISCUSSION THREAD

**Discussion by mrceperka** [tauri-apps/tauri#3291] [Q&A] [answered]
Hello :wave: ,

is there any way to make a windows bundle, while my host is linux?

Thanks for info :pray:

**Comment by FabianLars** [comment, marked as answer]
No there isn't and unfortunately won't be anytime soon (if at all). For now you need something like VMs or github actions (we provide a helper action here: https://github.com/tauri-apps/tauri-action).

Not asked for by you, but for mac specific additional reasoning, see here: https://github.com/tauri-apps/tauri/issues/279#issuecomment-660621484

**Reply by mrceperka** [reply]
Thanks for info :)

**Reply by mrceperka** [reply]
@FabianLars I've found this article - https://bevy-cheatbook.github.io/setup/cross/linux-windows.html. It's about cross compiling on linux, targetting windows.

Is there a way to make this work in tauri project?

**Reply by FabianLars** [reply]
So the gnu variant is out of the question anyway (doesn't work with the WebView bindings, other crates probably cause problems too, and the msvc toolchain is generally considered better (and it's the default))

The msvc variant is closer to producing a working binary. But it still won't work because the WebView bindings need Nuget support and i doubt that it works with that.
Even if it would work (and you end up with a working _binary_), you still can't build a MSI, because WiX doesn't work on Linux :/

**Reply by mrceperka** [reply]
Thanks for detailed explanation :heart:
